### PR TITLE
Clean the package

### PR DESCRIPTION
**Pull request by frankrousseau**
I made several operation to make the package cleaner: 
- Add comments
- Put information in the README
- Add Coffeelint to the depencies
- Clean the manifest
- Add lint command
- Linting

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/cozy/cozysdk-client"
+  },
+  "bugs": {
+    "url": "https://github.com/cozy/cozysdk-client/issues"
+  },
+  "main": "build/cozysdk-client.js",
+  "scripts": {
+    "build": "./node_modules/coffee-script/bin/coffee -cb --output lib src && browserify lib/index.js --debug --s cozysdk | exorcist dist/cozysdk-client.js.map > dist/cozysdk-client.js ",
```
Comment (aenario) [inline package.json]: I think the npm bash script are automatically executed with ./node_modules/.bin in their paths.

File: package.json [hunk 2]
```diff
@@ -1,27 +1,31 @@
...
+  },
+  "devDependencies": {
+    "coffee-jshint": "0.2.2",
+    "coffee-script": "1.8.0",
+    "coffeelint": "1.8.1",
+    "browserify": "13.0.0",
+    "exorcist": "0.4.0"
+  },
+  "dependencies": {
+    "coffeelint": "^1.8.1"
```
Comment (aenario) [inline package.json]: should be a devDependencies

File: src/index.coffee [hunk 3]
```diff
@@ -63,10 +76,13 @@ module.exports.updateAttributes = (docType, id, attributes, callback) ->
...
             callback new Error "Document #{id} not found"
         else if response.status isnt 200
-            callback new Error "#{response.status} -- #{body.id} -- Server error occured."
+            callback new Error \
+                "#{response.status} -- #{body.id} -- Server error occured."
         else
             callback null, JSON.parse body
 
+
+# Destryo the document that matches given doc type and given ID.
```
Comment (nono) [inline src/index.coffee]: s/Destryo/Destroy/

**Comment by nono** [comment]
Thanks @frankrousseau!

**Pull request merged**